tests/ui/test_ui.py

```diff
@@ -1,5 +1,10 @@
+from typing import Any, Callable, List, Optional, Tuple
+
 import pytest
+from pytest_mock import MockerFixture
+from urwid import Widget
```

Comment:
I'm not very confident in this being the most relevant type for the `widget` parameter in `widget_size`, does this fit?

Reply:
This seems fine for now - it is the base widget class after all?

Reply:
Yeah, verified by the class's docstring.